pkg/landscaper/webhook/add.go

```diff
@@ -0,0 +1,40 @@
+// SPDX-FileCopyrightText: 2020 SAP SE or an SAP affiliate company and Gardener contributors.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package webhook
+
+import (
+	"k8s.io/apimachinery/pkg/runtime"
+	"sigs.k8s.io/controller-runtime/pkg/log"
+	"sigs.k8s.io/controller-runtime/pkg/manager"
+
+	lscore "github.com/gardener/landscaper/pkg/apis/core"
+	"github.com/gardener/landscaper/pkg/utils/webhook"
+)
+
+var (
+	// DefaultAddOptions are the default AddOptions for AddToManager.
+	DefaultAddOptions = AddOptions{}
+)
+
+// AddOptions are options to apply when adding the AWS shoot webhook to the manager.
+type AddOptions struct{}
+
+var logger = log.Log.WithName("landscaper-validator")
+
+// AddToManager creates a webhook with the given options and adds it to the manager.
+func AddToManager(mgr manager.Manager) (*webhook.Webhook, error) {
+	logger.Info("Adding webhook to manager")
```

Comment:
```suggestion
// AddToManager creates a webhook with the given options and adds it to the manager.
func AddToManager(log logr.Logger, mgr manager.Manager) (*webhook.Webhook, error) {
	logger.Info("Adding webhook to manager")
```
we currently to not use global loggers but pass them when initializing.